inception-project/inception
# Bugfix/741 kbs sometimes used even though they are disabled

**What's in the PR**
* use only enabled KBs for annotations and the KB page
* warn the user if a disabled KB is set as scope for the concept feature

**How to test manually**
* check if warning is displayed properly when a disabled kb is set as scope for the concept feature
* check if bug has been fixed according to #741 

**Automatic testing**

**Documentation**
